justfile

```diff
@@ -4,7 +4,7 @@ install: install-contracts install-eip712sign
 # for any tasks run in production, a specific commit should be used for each dep.
 install-contracts:
   #!/usr/bin/env bash
-  forge install
+  forge install --shallow
```

Comment:
might speed up `forge install` significantly

Reply:
I would say we leave this alone for now and keep the plain `forge install`. I can't imagine this change being a problem, but we have a lot of moving parts with all the ongoing playbooks and we should minimize the diff/potential sources of error as much as possible, just to be safe